src/screens/Settings.js

```diff
@@ -74,8 +75,19 @@ class Settings extends React.Component {
    */
   handleReset = () => {
     $('#confirmResetModal').modal('hide');
+    // Remove from list of wallets
```

Comment:
This is pretty much the same code that we have in the `LockedWallet` file, maybe we could create a method in the wallet utils to do this, it just needs to receive the history to push.